memgpt/config.py

```diff
@@ -31,6 +31,120 @@ def set_field(config, section, field, value):
     config.set(section, field, value)
 
 
+@dataclass
+class Config:
```

Comment:
Ignore this, decided to just keep `MemGPTConfig` for now and maybe gradually migrate to this